third_party/libdav1d/BUILD.gn

```diff
@@ -118,7 +118,8 @@ if (is_win) {
     dav1d_copts += [ "/wd4028" ]
   }
 } else {
-  dav1d_copts += [ "-std=c99" ]
+  # Gets built with c99 upstream: https://source.corp.google.com/h/chrome-internal/codesearch/chrome/src/+/main:third_party/dav1d/BUILD.gn;l=94?q=c99%20lang:gn&ss=h%2Fchrome-internal%2Fcodesearch%2Fchrome%2Fsrc%2F%2B%2Frefs%2Fheads%2Fmain
```

Comment:
If it's getting built with c99 upstream, that's totally fine, we can leave it as is—we just want to have the default be c11

Reply:
gotcha, updated